build/Configuration.cs

```diff
@@ -0,0 +1,16 @@
+using System;
```

Comment:
We need to discourage R# from snipping redundant usings here otherwise PrettyBot ends up unhappy.

```suggestion
// ReSharper disable RedundantUsingDirective

using System;
```